_posts/2022-05-26-opensearch-2-0-is-now-available.md

```diff
@@ -0,0 +1,69 @@
+* **Java Jigsaw module support** means that with version 9.1, Lucene JARs are now proper Java modules, with module descriptors and dependency information. This aligns with the continuing evolution of OpenSearch to make the toolset more modular and extensible.  
+
+### **Document-level Alerting**
+
+**Document-level alerting** ([see GitHub issue](https://github.com/opensearch-project/alerting/issues/238)) allows users to create monitors that can generate alerts per document. Commonly used in security detection, these monitors use a similar approach as other types of alerting monitors available in OpenSearch: query level and bucket level. While those alerts use a summarized view of the data, document-level alerting can issue alerts on each document in the index, spotlighting which specific documents or records are triggering an alert in the monitor and avoiding monitoring gaps or data overlaps based on timestamps.
+
+![Image: Document-level alerting]({{ site.baseurl }}/assets/media/blog-images/2022-05-26-opensearch-2-0-is-now-available/docLevelMonitor.gif){: .img-fluid }
+
+### **Notifications**
+
+A new **Notifications plugin** ([see GitHub issue](https://github.com/opensearch-project/notifications/issues/181)) adds a unified notifications system to OpenSearch. Users no longer need to configure and manage notification channels for each plugin independently; with version 2.0, the Notifications plugin provides a centralized location to set up and manage notifications for relevant OpenSearch plugins. For example, in addition to managing notifications for the Alerting plugin, this plugin can generate a notification when a scheduled action is completed in the [Index State Management](https://opensearch.org/docs/latest/im-plugin/ism/index/) (ISM) plugin.
+
+![Image: Notifications plugin]({{ site.baseurl }}/assets/media/blog-images/2022-05-26-opensearch-2-0-is-now-available/notifications-short.gif){: .img-fluid }
+
+### **ML Commons Upgrades**
+
+Introduced with version 1.3, the **ML Commons plugin** ([see GitHub repo](https://github.com/opensearch-project/ml-commons) gains two new algorithms in version 2.0 to extend OpenSearch’s machine learning (ML) functionality to additional workloads, reduce the effort required to build ML features, and centralize computation, resource management, and security for ML processes. New algorithms for linear regression and localization join existing algorithms for kmeans and [Random Cut Forests](https://opensearch.org/blog/odfe-updates/2019/11/random-cut-forests/) to provide a comprehensive foundation for building and training ML models. The addition of linear regression aims to simplify development of ML models for predictive analysis; with localization, users can get a head start on developing ML approaches that reveal the key contributors to anomalies or any events that are detected, facilitating analyses and visualizations for root cause analysis and other use cases. 
+
+### Replacing Non-inclusive Terminology
```

Comment:
Do we need to include `Mapping types removal` as well ?

https://github.com/opensearch-project/OpenSearch/issues/2979

Reply:
Thanks @krisfreedain for the prompt response. 

Yes, there was an issue created for documentation https://github.com/opensearch-project/documentation-website/issues/429; which is already closed.